x/inflation/types/params.go

```diff
@@ -58,45 +62,19 @@ func DefaultParams() Params {
 		EpochIdentifier:         "day",                    // 1 day
 		ReductionPeriodInEpochs: 365,                      // 1 year
 		ReductionFactor:         sdk.NewDecWithPrec(5, 1), // 0.5
-		DistributionProportions: DistributionProportions{
-			Staking:          sdk.NewDecWithPrec(4, 1),  // 0.4
-			PoolIncentives:   sdk.NewDecWithPrec(25, 2), // 0.25
-			DeveloperRewards: sdk.NewDecWithPrec(25, 2), // 0.25
-			CommunityPool:    sdk.NewDecWithPrec(1, 1),  // 0.1
+		AllocationProportions: AllocationProportions{
+			StakingRewards:  sdk.NewDecWithPrec(4, 1),  // 0.4
+			TeamVesting:     sdk.NewDecWithPrec(25, 2), // 0.25
+			UsageIncentives: sdk.NewDecWithPrec(25, 2), // 0.25
+			CommunityPool:   sdk.NewDecWithPrec(1, 1),  // 0.1
 		},
-		WeightedDeveloperRewardsReceivers:    []WeightedAddress{},
-		MintingRewardsDistributionStartEpoch: 0,
-	}
-}
-
-// validate params
-func (p Params) Validate() error {
-	if err := validateMintDenom(p.MintDenom); err != nil {
-		return err
-	}
-	if err := validateGenesisEpochProvisions(p.GenesisEpochProvisions); err != nil {
-		return err
-	}
-	if err := epochtypes.ValidateEpochIdentifierInterface(p.EpochIdentifier); err != nil {
-		return err
-	}
-	if err := validateReductionPeriodInEpochs(p.ReductionPeriodInEpochs); err != nil {
-		return err
+		TeamVestingProvision: sdk.NewCoin(
+			sdk.DefaultBondDenom,
```

Comment:
this should be `"aevmos"` instead of `stake`

Reply:
@fedekunze you mean hardcoded? And should `"aevmos"` also be set for the MintDenom then?